_data/blogs.yml

```diff
@@ -203,4 +203,9 @@
   img: DSC_3288.jpg
   author_name: Aman Singh
   
+  - url: https://hiteshsgci2017experience.wordpress.com/
```

Comment:
It is not according to the format specified in the readme.md
Remove the space that you have provided before `-`